src/VisualStudio/Core/Def/Options/VisualStudioOptionPersisterProvider.cs

```diff
@@ -29,7 +29,7 @@ internal sealed class VisualStudioOptionPersisterProvider : IOptionPersisterProv
         // maps config name to a read fallback:
         private readonly ImmutableDictionary<string, Lazy<IVisualStudioStorageReadFallback, OptionNameMetadata>> _readFallbacks;
 
-        private VisualStudioOptionPersister? _lazyPersister;
+        private readonly Threading.AsyncLazy<IOptionPersister> _lazyPersister;
```

Comment:
`Threading.AsyncLazy<T>` is JTF aware. Eventually Roslyn should switch to it uniformly, but in the meantime Roslyn code  with UI thread dependencies should be using it.